### PR TITLE
filter out JSON API entities with broken relations

### PR DESCRIPTION
**Pull request by david-kubecka**
JIRA: RAIL-3248

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: libs/sdk-backend-tiger/src/backend/workspace/dashboards/index.ts [hunk 1]
```diff
@@ -37,6 +37,7 @@ import {
 } from "../../../convertors/toBackend/AnalyticalDashboardConverter";
 import { TigerAuthenticatedCallGuard } from "../../../types";
 import { objRefToIdentifier } from "../../../utils/api";
+import { ValidateRelationsHeader } from "@gooddata/api-client-tiger/dist/metadataUtilities";
```
Comment (no23reason) [inline libs/sdk-backend-tiger/src/backend/workspace/dashboards/index.ts:40]: Please avoid importing from `dist`, `import { ValidateRelationsHeader } from "@gooddata/api-client-tiger";` should work too. This is repeated in all the changed files.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3168/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4811/ : SUCCESS in 12m 29s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3169/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4812/ : SUCCESS in 12m 20s

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1237

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/942/ : SUCCESS in 54s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1211/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4813/ : SUCCESS in 12m 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/326/ : SUCCESS in 8m 10s (non-voting)

**Pull request merged**